packages/apps/docs/src/components/Layout/components/Aside/ListItem.tsx

```diff
@@ -53,14 +55,15 @@ export const ListItem: FC<IProps> = ({
     >
       {item.children.length > 0 && (
         <AsideList inner={true}>
-          {item.children.map((innerItem) => {
+          {item.children.map((innerItem, index) => {
```

Comment:
hm...can we maybe do this calculation in the remark script?
it makes the react code a bit harder to read.